Makefile.template

```diff
@@ -139,7 +139,7 @@ endif
 	touch $(TIMESTAMP_CONFIGURE)
 
 build_internal: $(BUILD_FOLDER)
-	NNFW_WORKSPACE="$(WORKSPACE)" NPROCS="$(NPROCS)" PARALLEL_BUILD="1" ./nnfw build
+	NNFW_WORKSPACE="$(WORKSPACE)" NPROCS="$(NPROCS)" PARALLEL_BUILD="$(PARALLEL_BUILD)" ./nnfw build
```

Comment:
Note https://github.com/Samsung/ONE/blob/d85b609ecd4b17f67ecb94d3500f68a0c58c5a86/Makefile.template#L7

Reply:
Also note, from my best knowledge, we have no CI script using `PARALLEL_BUILD` configured explicitly